_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
 
 
-### babelfishpg_tsql.ansi_nulls
+### `babelfishpg_tsql.ansi_nulls`
 
-Specifies ISO compliant behavior of the "equal" (`=`) and "not equal" (`<>`) comparison 
+Specifies ISO compliant behavior of the "equal" (`=`) and "not equal" (`<>`) comparison
 operators when they are used with `NULL` values.
 
 
-### babelfishpg_tsql.ansi_padding
+### `babelfishpg_tsql.ansi_padding`
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.ansi_warnings
+### `babelfishpg_tsql.ansi_warnings`
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.arithabort
+### `babelfishpg_tsql.arithabort`
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.arithignore
+### `babelfishpg_tsql.arithignore`
 
-`ON` setting is not allowed for this option. Please use 
+The setting `ON` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.concat_null_yields_null
+### `babelfishpg_tsql.concat_null_yields_null`
 
 If enabled, concatenating a `NULL` value produces a `NULL` result.
 
 
-### babelfishpg_tsql.cursor_close_on_commit
+### `babelfishpg_tsql.cursor_close_on_commit`
 
-`ON` setting is not allowed for this option. Please use 
+The setting `ON` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.database_name
+### `babelfishpg_tsql.database_name`
 
 Predefined Babelfish database name. The default value is `babelfish_db`.
 
 
-### babelfishpg_tsql.datefirst
+### `babelfishpg_tsql.datefirst`
 
 Sets the first day of the week to a number from 1 through 7.
 
 
-### babelfishpg_tsql.default_locale
+### `babelfishpg_tsql.default_locale`
 
 The default locale to use when creating a new collation. The default value is
-`en_US`. Only superusers can set this variable. 
+`en_US`. Only superusers can set this variable.
 
 
-### babelfishpg_tsql.disable_batch_auto_commit
+### `babelfishpg_tsql.disable_batch_auto_commit`
 
 Disable auto commit inside procedures. The default value is `false`.
 
 
-### babelfishpg_tsql.disable_internal_savepoint
+### `babelfishpg_tsql.disable_internal_savepoint`
 
 Disable internal savepoints. The default value is `false`.
 What is the purpose of this variable? It controls whether a savepoint is created to
 handle undo if a command fails. Savepoints are not started for batch commands, as
 error handling must be taken care of at the statement level.
```

Comment:
This should not be documented! this is a development-only setting, with dramatic consequences when it is used, Best remove this. If not, make very clear that this should not be used by customers in any situation.

Reply:
I have added the warning.

Off topic, but does Babelfish create a savepoint for every statement? If yes, that would have severe performance impacts...